.github/workflows/build-node-packages.yml

```diff
@@ -0,0 +1,202 @@
+name: Generate Node.js
+on:
+# TODO: currently workflow dispatch endpoint does not work. I will investigate
+  workflow_dispatch:
+    inputs:
+      VERSION:
+        description: 'Node version to build and upload'
+        required: true
+        default: '14.2.0'
+      PUBLISH_RELEASES:
+        description: 'Whether to publish releases'
+        required: true
+        default: 'false'
+
+env:
+  VERSION: ${{ github.event.inputs.VERSION }}
+  ARCHITECTURE: x64
+
+jobs:
+  build_node:
+    name: Build Node ${{ github.event.inputs.VERSION }} ${{ matrix.platform }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [linux, darwin, win32]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Create artifact directories
+      run: |
+        binariesDirectory=$RUNNER_WORKSPACE/binaries
```

Comment:
Why we need to create this directories here and define new environment variables?
Let's do that in `build-node.ps1`

Reply:
done